1/7_11718_pjs.cpp

```diff
@@ -0,0 +1,41 @@
+#include "1.h"
+#include <cstring>
+#include <string>
+
+using namespace std;
+
+std::string trim(const std::string& str,
+	const std::string& whitespace = " \t")
+{
+	const auto strBegin = str.find_first_not_of(whitespace);
+	if (strBegin == std::string::npos)
+		return ""; // no content
+
+	const auto strEnd = str.find_last_not_of(whitespace);
+	const auto strRange = strEnd - strBegin + 1;
+
+	return str.substr(strBegin, strRange);
+}
+

```

Comment:
This trim function is overrided ?

Reply:
This trim function is maked by other people 
please, approve this PR